Gems/Archive/Code/Platform/Mac/PAL_mac.cmake

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+set(PAL_TRAIT_ARCHIVE_SUPPORTED TRUE)
+set(PAL_TRAIT_ARCHIVE_TEST_SUPPORTED FALSE)
+set(PAL_TRAIT_ARCHIVE_EDITOR_TEST_SUPPORTED FALSE)
```

Comment:
Is this planned for the future?

Reply:
Yes. 
Test will be added as functionality is added to the gem.